libs/partners/astradb/tests/integration_tests/vectorstores/test_astradb.py

```diff
@@ -0,0 +1,867 @@
+"""
+Test of Astra DB vector store class `AstraDBVectorStore`
+
+Required to run this test:
+    - a recent `astrapy` Python package available
+    - an Astra DB instance;
+    - the two environment variables set:
+        export ASTRA_DB_API_ENDPOINT="https://<DB-ID>-us-east1.apps.astra.datastax.com"
+        export ASTRA_DB_APPLICATION_TOKEN="AstraCS:........."
```

Comment:
add necessary secrets to `.github/workflows/_release.yml` and `.github/workflows/_integration_test.yml`